tests/e2e/tests/devfiles/Scala.spec.ts

```diff
@@ -29,18 +28,12 @@ const compileTaskkName: string = 'sbt compile';
 const runTaskName: string = 'sbt run';
 const testTaskName: string = 'sbt test';
 const stack: string = 'Scala';
-let workspaceName: string;
 
 // skipping scala to enable pre-release suite to be easily used for updates until https://github.com/eclipse/che/issues/18662 is fixed
-suite.skip(`${stack} test`, async () => {
+suite(`${stack} test`, async () => {
```

Comment:
method comment looks outdated

Reply:
fixed. forgot to disable it after testing